<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotWidget</class>
 <widget class="QWidget" name="RobotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>75</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblPixmap">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblRobot">
       <property name="text">
        <string>Robot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveRobot">
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblSensingRangeBuddy">
       <property name="text">
        <string>Sensing range:</string>
       </property>
       <property name="buddy">
        <cstring>sbSensingRange</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbSensingRange">
       <property name="suffix">
        <string> m</string>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>50.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkFillSensingRange">
     <property name="text">
      <string>fill sensed area</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
